src/ripple/protocol/Rate.h

```diff
@@ -40,10 +41,10 @@ struct Rate
 
     Rate () = default;
 
+    explicit
     Rate (std::uint32_t rate)
```

Comment:
Classes that emulate built-in numerics should adopt the default construction behavior of built-in numerics. They should be left uninitialized.